### DISCUSSION TITLE
Short Buttons

### DISCUSSION THREAD

**Discussion by TeaqariaWTF** [anddea/revanced-patches#705] [Q&A]
Hi,

I want to change these buttons, can you help me with this?

![Screenshot_20240707-131923_YouTube](https://github.com/anddea/revanced-patches/assets/90970076/8ab8fdcc-e90b-4f72-aa2d-8d053529c3e8)

**Comment by anddea** [comment]
You need to elaborate. There are other options for this in the Manager.

**Reply by TeaqariaWTF** [reply]
I am using a tool to create modules for Magisk on Github.

Which settings should I include?

**Reply by anddea** [reply]
Change IconType in options.json to one of these keys:

https://github.com/anddea/revanced-patches/blob/98b26f1202da953dfe5afa5adbce80befd132ad9/src/main/kotlin/app/revanced/patches/youtube/layout/actionbuttons/ShortsActionButtonsPatch.kt#L26-L30

**Reply by TeaqariaWTF** [reply]
`[
  {
    "patchName": "Custom branding icon YouTube",
    "options": [
      {
        "key": "AppIcon",
        "value": "MMT"
      }
    ]
  },
  {
    "patchName": "Custom branding name YouTube",
    "options": [
      {
        "key": "AppName",
        "value": "YouTube"
      }
    ]
  },
  {
    "patchName": "Custom branding icon YouTube Music",
    "options": [
      {
        "key": "SplashHeaderIcon",
        "value": "MMT"
      },
      {
        "key": "AppIcon",
        "value": "MMT"
      }
    ]
  },
  {
    "patchName": "Custom branding name YouTube Music",
    "options": [
      {
        "key": "AppNameNotification",
        "value": "YouTube Music"
      },
      {
        "key": "AppNameLauncher",
        "value": "YouTube Music"
      }
    ]
  },
  {
    "patchName": "Custom Shorts action buttons",
    "options": [
      {
        "key": "IconType",
        "value": "OutlineCircle"
      }
    ]
  }
]`


![Screenshot_20240707-154703_Iceraven](https://github.com/anddea/revanced-patches/assets/90970076/84358ae8-c140-4a1a-8f76-d09fff88bc2a)

**Reply by TeaqariaWTF** [reply]
Are my settings correct?

**Reply by anddea** [reply]
There's no Short overlay buttons patch in the latest dev. There's custom shorts action buttons patch. Delete your old options.json.

**Reply by TeaqariaWTF** [reply]
Hi , 
I need help again, can you help me?

This is how I set options.js

`[
  {
    "patchName": "Custom branding icon YouTube",
    "options": [
      {
        "key": "AppIcon",
        "value": "MMT"
      }
    ]
  },
  {
    "patchName": "Custom branding name YouTube",
    "options": [
      {
        "key": "AppName",
        "value": "YouTube"
      }
    ]
  },
  {
    "patchName": "Custom branding icon YouTube Music",
    "options": [
      {
        "key": "SplashHeaderIcon",
        "value": "MMT"
      },
      {
        "key": "AppIcon",
        "value": "MMT"
      }
    ]
  },
  {
    "patchName": "Custom branding name YouTube Music",
    "options": [
      {
        "key": "AppNameNotification",
        "value": "YouTube Music"
      },
      {
        "key": "AppNameLauncher",
        "value": "YouTube Music"
      }
    ]
  },
  {
    "patchName": "Custom Shorts action buttons",
    "options": [
      {
        "key": "IconType",
        "value": "OriginalOutline"
      }
    ]
  }
]`

But it doesn't add the icon I want

**Reply by anddea** [reply]
`Custom branding icon YouTube` is now `Custom branding icon for YouTube`, same for YouTube Music. It's better to ask such questions on Reddit or Telegram support groups.
